packages/yum/test.sh

```diff
@@ -63,9 +63,10 @@ echo "::group::Install packages for test"
 
 case ${os} in
   amazon-linux)
-    ${DNF} install -y \
-           libpq-devel \
-           postgresql-server-devel
+    if [ "$((${postgresql_version} != 13))" -eq 1 ]; then
```

Comment:
Can we simplify this?

```suggestion
    if [ ${postgresql_version} -ne 13 ]; then
```

Reply:
`if [ ${postgresql_version} -ne 13 ]` throw an error like `[: hoge: integer expression expected` if `${postgresql_version}` is not a number. `if [ "$((${postgresql_version} != 13))" -eq 1 ]` does not throw any error in that case.

`${postgresql_version}` seems always a number, though... (https://github.com/pgroonga/pgroonga/blob/eaecaee69caf44ff42ea5f46165cc8cea1496984/packages/yum/test.sh#L48).

I think this is not a major problem because in that case this script may not work correctly if this is `if [ "$((${postgresql_version} != 13))" -eq 1 ]`.
However, current  implementation use this expression in case checking `${postgresql_version}`, I have followed them.

https://github.com/pgroonga/pgroonga/blob/eaecaee69caf44ff42ea5f46165cc8cea1496984/packages/yum/test.sh#L131

Reply:
If we expect `postgresql_version` is a number, we should get an error when the expectation isn't satisfied. We should not ignore the unexpected case silently to miss a problem.

Reply:
In general, improving maintainability is important than keeping existing style. Consistency is one of maintainability points but simplicity is also one of maintainability points.

Reply:
> In general, improving maintainability is important than keeping existing style. Consistency is one of maintainability points but simplicity is also one of maintainability points.

I see. Thanks!!

I have fixed them.